src/js/popout.js

```diff
@@ -0,0 +1,10 @@
+import M from 'materialize-css';
+
+document.addEventListener('DOMContentLoaded', function() {
+    var elems = document.querySelectorAll('.collapsible');
+    var instances = M.Collapsible.init(elems,  {
```

Comment:
Remove `var instances =`
Since this is react you can place this code in `ComponentDidMount` function
Have a look at:

1. https://github.com/isohack/hackathon-webapp/blob/master/src/components/landingPage.jsx#L4
2. https://github.com/isohack/hackathon-webapp/blob/master/src/components/landingPage.jsx#L34